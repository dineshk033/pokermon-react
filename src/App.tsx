import React from 'react';
import './App.css';
import { Typography } from '@mui/material';
function App() {
  return (
    <div className="test">
      <Typography>Project Initial setup</Typography>
    </div>
  );
}

export default App;
