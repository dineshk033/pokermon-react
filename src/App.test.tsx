import React from 'react';
import { shallow } from 'enzyme';
import App from './App';

test('renders learn react link', () => {
  const wrrapper = shallow(<App/>);
 console.log(wrrapper)
  expect(wrrapper).toMatchSnapshot();
});
