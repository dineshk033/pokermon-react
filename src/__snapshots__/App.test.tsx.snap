// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders learn react link 1`] = `
<div
  className="test"
>
  <ForwardRef(Typography)>
    Project Initial setup
  </ForwardRef(Typography)>
</div>
`;
